conans/server/conf/default_server_conf.py

```diff
@@ -22,6 +22,22 @@
 disk_authorize_timeout: 1800
 updown_secret: {updown_secret}
 
+# If you want to use a different authenticator that the default based in the [users] list
```

Comment:
I would just add a link to the docs, or just "check the docs.conan.io". 
Not relevant for most people